src/sss_client/ssh/sss_ssh_knownhosts.c

```diff
@@ -0,0 +1,211 @@
+/*
+    Authors:
+        Alejandro Lopez <allopez@redhat.com>
+
+    Copyright (C) 2024 Red Hat, Inc.
+
+    This program is free software; you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation; either version 3 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#define _GNU_SOURCE    /* For basename(3) */
```

Comment:
Imo, it's better (more consistent) to include `config.h`

Reply:
Actually this is no longer needed as `basename()` is no longer used.

removed.